docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -28,17 +28,267 @@ This document covers a range of common scenarios using the PXIe-148X Generation
+        >
+        > The **GPIO to Display** control specifies GPIO lines to display on the GPIO Timestamps Waveform after the acquisition completes. Timestamps for GPIO lines not included in the **GPIO to Display** array are logged, but not displayed.
+
+    - Select **SO0 Serializer** from the **GPIO Bank Select** drop down menu.
+    
+        > The **GPIO Bank Select** control specifies the GPIO bank used for the **GPIO Bank Output**, **GPIO Bank Output Enable** and **GPIO Bank Read** controls during the generation. The **GPIO Bank Select** selection may be changed at runtime.
+
+    - In the **GPIO Bank Output Enable** cluster, enable the **GPIO 0** control.
+        > The **GPIO Bank Output Enable** controls are used as write enables for manual GPIO writes to override the defined GPIO routing. In this case we are choosing to write to line 0 of the GPIO selected on the **GPIO Bank Select** control and leave all other lines in the bank unchanged.
+
+   > The settings on the **GPIO** tab are now similar to the image below.
+   
+   ![Manually Reading and Writing to GPIO - GPIO Tab Settings](../../images/PXIe-148X-Gen-GPIO-GPIOTabSettings.png)
+
+2. Run the VI, wait for the **Waiting for Serializer Setup** indicator to illuminate, and click the **Serializer Setup Complete** control button to start the generation.
+
+3. Click **Write GPIO** to write a false value to line 0 on the SO 0 Serializer GPIO bank.
+    > Notice that the **GPIO 0** Boolean indicator in the **GPIO Bank Read** cluster is not illuminated (false).
+
+4. Set the **GPIO 0** button in the **GPIO Bank Write** control to <font face = "courier new">true</font> then click the **Write GPIO** button to write a true value to line 0 on the SO 0 Serializer GPIO bank.
+    > Notice that the **GPIO 0** Boolean indicator in the **GPIO Bank Read** cluster is illuminated (true).
+
+5. Set the **GPIO 0** button in the **GPIO Bank Write** control to <font face = "courier new">false</font> then click the **Write GPIO** button to write a false values to line 0 on the SO 0 Serializer GPIO bank.
+    > Notice that the **GPIO 0** Boolean indicators in the **GPIO Bank Read** cluster are not illuminated (false).
+
+6. Click the **Stop Generation** button.
+
+7. Once the VI stops running, click on the **GPIO Timestamps** tab to view a digital waveform of the selected GPIO line(s).
+
+    > GPIO timestamp data is displayed in the **GPIO Timestamps Waveform** digital waveform indicator. The pattern observed matches the sequence of manual writes that were performed on GPIO line 0 during the tutorial.
+    
+    > Note: The digital waveform is read from the GPIO_Timestamps.tdms file and filtered to display only timestamps for GPIO lines included in the **GPIO to Display** array. The **GPIO Timestamps Waveform** display is updated after the acquisitions completes.
```

Comment:
update to: "after the generation completes"

Reply:
Done.